mysql-test/suite/rocksdb/t/index_merge_rocksdb.test

```diff
@@ -0,0 +1,93 @@
+--source include/have_rocksdb.inc
+# Skiping this test from Valgrind execution as per Bug-14627884
+--source include/not_valgrind.inc
+# Adding big test option for this test.
+#--source include/big_test.inc
```

Comment:
I'll remove these lines. They were left over from the earlier test cases which have been removed.